components/token-operation/ConfirmTokenOperationStep.tsx

```diff
@@ -15,18 +15,30 @@ export const ConfirmTokenOperationStep = ({
   onBack?: () => void;
 }) => {
   const navigate = useNavigate();
+  const { fetchTokenInfo } = useKasplex();
   const { watch } = useFormContext<TokenOperationFormData>();
+  const [mintAmount, setMintAmount] = useState<string>();
   const opData = watch("opData");
-  const decimalCoefficient = Math.pow(
-    10,
-    opData.dec ? parseInt(opData.dec, 10) : 8,
-  );
+  const decimal = applyDecimal(opData.dec);
   const kapsaPrice = useKaspaPrice();
 
   const onClose = () => {
     navigate("/dashboard");
   };
 
+  useEffect(() => {
+    if (opData.op === "mint") {
+      fetchTokenInfo(opData.tick).then((tokenInfo) => {
+        const tokenDetails = tokenInfo?.result?.[0];
+        if (!tokenDetails) {
+          return;
+        }
+
+        setMintAmount(decimal(parseInt(tokenDetails.lim, 10)).toLocaleString());
+      });
```

Comment:
![image](https://github.com/user-attachments/assets/638975ed-d27d-4f3a-bf82-5b4f249f17a4)
There is no `lim` inside `tokenInfo` endpoint:
```
{
    "message": "successful",
    "prev": "YYYY",
    "next": "TTTT",
    "result": [
        {
            "tick": "YYYY",
            "balance": "990",
            "locked": "0",
            "dec": "8",
            "opScoreMod": "861158350000"
        },
        {
            "tick": "TTTT",
            "balance": "299999983400",
            "locked": "0",
            "dec": "8",
            "opScoreMod": "865571120000"
        }
    ]
}
```

Reply:
23c79e872e35c519e6cc2b06e5921faf00be7d79

Has been fixed when I replaced `fetch` with `useSWR`.